recipe/meta.yaml

```diff
@@ -8,45 +8,49 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dask_sql-{{ version }}.tar.gz
-  sha256: edb59cf82699f2fd260335641fd56b9b60932db39ee6a3cff3b0a720ca58578a
+  sha256: d34d0f48c4989cf1cd33d3379d7b138e204164becb8bbf0b15677f6ade5dd374
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: true   # [py2k]
+  entry_points:
+    - dask-sql-server = dask_sql.server.app:main
+    - dask-sql = dask_sql.cmd:main
+  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
-  build:
-    - maven >=3.6.0
   host:
     - pip
-    - python >=3.6
-    - setuptools_scm
+    - python
+    - setuptools-scm
   run:
-    - python >=3.6
-    - dask >=2.19.0,!=2021.3.0
+    - python
+    - dask ==2021.10.0
     - pandas >=1.0.0
     - jpype1 >=1.0.2
-    - openjdk >=8
```

Comment:
Obviously we want to keep this. Raised issue ( https://github.com/conda-incubator/grayskull/issues/261 )

Reply:
Do we also want to keep the `maven` build requirement?

Reply:
Sorry missed that. Agree we want to keep that. Commented above